.circleci/config.yml

```diff
@@ -12,7 +12,7 @@ defaults:
       - image: 'quay.io/influxdb/telegraf-ci:1.15.5'
   mac: &mac
     macos:
-      xcode: 11.3.1
+      xcode: 12.2.0
```

Comment:
Playing around with circleci to make sure I understand it - this bumps the macos version (see commit message).